File: MVCExe/Models/Customer.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Data.Entity.Spatial

Partial Public Class Customer
    <StringLength(4)>
    Public Property CustomerID As String

    <StringLength(50)>
    Public Property CustomerName As String

    <StringLength(2)>
    Public Property MemberCategory As String

    Public Property Age As Integer?

    Public Property AmountSpent As Decimal?

    <StringLength(65)>
    Public Property Address As String

    <StringLength(30)>
    Public Property City As String

    <StringLength(4)>
    Public Property OldPostalCode As String

    <StringLength(6)>
    Public Property PostalCode As String

    <StringLength(3)>
    Public Property CountryCode As String

    <StringLength(2)>
    Public Property ContactTitle As String

    <StringLength(9)>
    Public Property PhoneNumber As String

    <StringLength(5)>
    Public Property Extension As String

    <StringLength(30)>
    Public Property FaxNumber As String

    <StringLength(50)>
    Public Property EmailAddress As String

    <StringLength(255)>
    Public Property Notes As String
End Class

File: MVC_DEMO/Models/Customer.cs
namespace MVC_DEMO.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    public partial class Customer
    {
        [StringLength(4)]
        public string CustomerID { get; set; }

        [StringLength(50)]
        public string CustomerName { get; set; }

        [StringLength(2)]
        public string MemberCategory { get; set; }

        public int? Age { get; set; }

        public decimal? AmountSpent { get; set; }

        [StringLength(65)]
        public string Address { get; set; }

        [StringLength(30)]
        public string City { get; set; }

        [StringLength(4)]
        public string OldPostalCode { get; set; }

        [StringLength(6)]
        public string PostalCode { get; set; }

        [StringLength(3)]
        public string CountryCode { get; set; }

        [StringLength(2)]
        public string ContactTitle { get; set; }

        [StringLength(9)]
        public string PhoneNumber { get; set; }

        [StringLength(5)]
        public string Extension { get; set; }

        [StringLength(30)]
        public string FaxNumber { get; set; }

        [StringLength(50)]
        public string EmailAddress { get; set; }

        [StringLength(255)]
        public string Notes { get; set; }
    }
}
